test/artifact/inspec_env_test.rb

```diff
@@ -0,0 +1,7 @@
+require_relative "artifact_helper"
+
+class TestInspecEnv < ArtifactTest
+  def test_env
+    assert_artifact(:env)
```

Comment:
This fails on windows, so should be skipped for now and re-written to expect the failure.